gradlew

```diff
@@ -89,7 +90,7 @@ location of your Java installation."
 fi
 
 # Increase the maximum file descriptors if we can.
-if [ "$cygwin" = "false" -a "$darwin" = "false" -a "$nonstop" = "false" ] ; then
+if [ "$cygwin" = "false" -a "$darwin" = "false" ] ; then
```

Comment:
So this came as part of the initial defaults. I guess unless we have no reason to add it back, we can leave it this way. Ack.